src/components/AccountConfigDialog/DisconnectAccountConfigDialog.jsx

```
@@ -237,22 +235,15 @@ const Run = ({
 }) => {
   useEffect(() => {
     let timer = null;
```

ESLint

[error] 'timer' is never reassigned. Use 'const' instead.

(prefer-const)